exporter/awsxrayexporter/internal/translator/segment_test.go

```diff
@@ -1406,7 +1397,7 @@ func validateLocalRootWithoutDependency(t *testing.T, segment *awsxray.Segment,
 	assert.Equal(t, expectedTraceID, *segment.TraceID)
 	assert.Equal(t, "POST", *segment.HTTP.Request.Method)
 	assert.Equal(t, 2, len(segment.Annotations))
-	assert.Equal(t, "myRemoteService", segment.Annotations["aws_remote_service"])
+	assert.Nil(t, segment.Annotations["aws_remote_service"])
```

Comment:
Why did this change from an `equal` to a `nil`? Did this annotation disappear?

Reply:
Yes. The input key is "aws.remote.service", so originally it convert to aws_remote_service, so aws_remote_service now from "myRemoteService" to Nil, aws.remove.service from Nil to "myRemoteService", please refer to line 1101